### ISSUE TITLE
Different Result

### ISSUE THREAD

**Issue by ShirleyHan6** [EdinburghNLP/Refresh#23]
Hi Shashi,

I also use pyrouge (https://github.com/bheinzerling/pyrouge) to calculate the ROUGE score and I use exactly the same code as you do (in reward_utils.py). 

However, the result I've gotten with the test set is a lot higher than yours. 

Did you do any preprocessing before summarizing on the test set, or what do you think could cause me to get a higher score?

(I am using python2.7 and tensorflow 0.12)

Thank you very much!
Simeng

**Comment by ShirleyHan6** [comment]
I found an error. pyrouge treats punctuations as a word and we must eliminate the punctuations in the test set before running the ROUGE test. However, my result is still higher.

**Comment by shashiongithub** [comment]
Could you use the Pyrouge and let me know if you still get a different result? I am not doing anything other than also used for training data and reported in the paper.

**Comment by ShirleyHan6** [comment]
I used pyrouge and, without doing anything on the data provided in this repo, got the same result. 

Thanks!

**Comment by FaridaaAli** [comment]
@ you mean that you got the same rouge scores ?? i upgraded tf to tf =10 and python = 3.6 and try to run the code as it is with the same data but i did not get the same results.

could you suggest me anything ?
